sbin/build-redisjson

```diff
@@ -54,22 +54,27 @@ $OP cd RedisJSON
 runn git checkout $BRANCH
 runn git pull --quiet --recurse-submodules
 
-# Temporary patch to use a fixed GOOD_NIGHTLY version because of a bug in the 
-# latest nightly https://github.com/rust-lang/rust/issues/125319
-sed -i 's/# GOOD_NIGHTLY=.*/GOOD_NIGHTLY=nightly-2024-05-16/' ./deps/readies/bin/getrust
 
 if is_command python3; then
 	runn python3 -m pip install virtualenv
 	runn python3 -m virtualenv venv
 	. venv/bin/activate
 fi
-# Install only rust, because the rest of the dependencies are already installed
-runn $READIES/bin/getrust
-$OP source /etc/profile.d/rust.sh
 
-runn BINROOT=$TARGET_DIR make SAN=$SAN
+# Install only rust, because the rest of the dependencies are already installed
+if [[ $NOP != 1 ]]; then
+	runn "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y"
```

Comment:
Great. Can we also not use `runn` (a readies command) to complete the removal of readies here?

Reply:
OK. Done. Thanks.